odl/solvers/functional/functional.py

```diff
@@ -1343,43 +1339,37 @@ class BregmanDistance(Functional):
 
     Notes
     -----
-    Given a functional :math:`f`, which has a (sub)gradient :math:`\partial f`,
-    and given a point :math:`y`, the Bregman distance functional
-    :math:`D_f(\cdot, y)` in a point :math:`x` is given by
+    Given a functional :math:`f`, a point :math:`y`, and a (sub)gradient
+    :math:`p \in \partial f(y)`, the Bregman distance functional
+    :math:`D_f^p(\cdot, y)` in a point :math:`x` is given by
 
     .. math::
-        D_f(x, y) = f(x) - f(y) - \\langle \partial f(y), x - y \\rangle.
+        D_f^p(x, y) = f(x) - f(y) - \langle p, x - y \rangle.
 
 
-    For mathematical details, see `[Bur2016]`_.
+    For mathematical details, see
+    `[Bur2016] <https://arxiv.org/abs/1505.05191>`_. See also the Wikipedia
+    article: https://en.wikipedia.org/wiki/Bregman_divergence
 
     References
     ----------
-    Wikipedia article: https://en.wikipedia.org/wiki/Bregman_divergence
-
     [Bur2016] Burger, M. *Bregman Distances in Inverse Problems and Partial
     Differential Equation*. In: Advances in Mathematical Modeling, Optimization
     and Optimal Control, 2016. p. 3-33.
-
-    .. _[Bur2016]:  https://arxiv.org/abs/1505.05191
     """
 
-    def __init__(self, functional, point, func_subgrad=None):
+    def __init__(self, functional, point, subgrad=None):
```

Comment:
I think the `=None` should be removed, no?

Reply:
Correct, I overlooked that